index.html

```diff
@@ -113,21 +113,22 @@ <h3 class="inter">Skills</h3>
     <h2 class="title inter naval">I'm always interested in hearing about new projects, so if you'd like to chat please
       get in
       touch.</h2>
-    <form action="https://formspree.io/f/xjvloqrk" method="POST" class="form-container">
+    <form action="https://formspree.io/f/xjvloqrk" method="POST" class="form-container" name="contact-form" novalidate>
       <ul>
         <li>
           <input type="text" id="name" class="inter" aria-label="Name" placeholder="Full name" maxlength="30"
             name="name" required>
         </li>
         <li>
           <input type="email" id="email" class="inter" aria-label="Email" placeholder="Email address" name="email"
-            required>
+            required pattern="[^A-Z]+">
```

Comment:
Fixed. Sorry, I implemented JavaScript validation for the email field only as per project requirements. But since I used the novalidate flag in the form, it wasn't validated through the built-in API.

Thank you for your feedback. Now I implemented a nice validation for all of them with user feedback.